lib/atom-notes.js

```diff
@@ -18,6 +18,7 @@ export default {
    * @param {SerializedIndex} state A previously serialized document storage index to load.
    */
   activate (state) {
+    console.info("Activating atom-notes")
```

Comment:
I use eslint to ensure that the code meets a style standard. Part of that standard is that we use single quotes for strings instead of double quotes. Would you mind changing these `"` to `'`?